packages/neutrino-middleware-eslint/README.md

```diff
@@ -127,6 +127,19 @@ pkg.neutrino.use
 module.exports = api.eslintrc();
 ```
 
+Projects authors may also face the problem when their code editor lints all files in the project and highlights errors in all of them. It ignores Neutrino `include` and `exclude` options. This is because editors use CLI to call ESLint instead of Neutrino infrastructure.
```

Comment:
> Projects may face a problem when their editor or IDE lints all files and highlights errors that were normally excluded from source, i.e. Neutrino's `include` and `exclude` options. This is because the ESLint CLI does not have a way to specify included and excluded files from configuration. If you still wish to use ESLint's CLI for linting, consider setting [CLI flags](http://eslint.org/docs/user-guide/command-line-interface#options) or using an [eslintignore](http://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories) to choose which files to include or exclude from linting.